source/UIAHandler/__init__.py

```diff
@@ -934,5 +937,25 @@ def isNativeUIAElement(self,UIAElement):
 		return False
 
 
+handler: Optional[UIAHandler] = None

```

Comment:
This, and the initialization code, might be better suited to the top of file

Reply:
I agree it would be nicer. The only problem you may encounter would be a Linter complain about `UIAHandler` not being defined yet. This can be easily worked around either by:
- placing `UIAHandler` in quotes or
- moving `UIAHandler` to a separate file, though this should happen in a follow up.